events.js:*
      throw er; // Unhandled 'error' event
      ^

Error: foo:bar
    at bar (*test-unhandled-error-stack.js:*:*)
    at foo (*test-unhandled-error-stack.js:*:*)
    at Object.<anonymous> (*test-unhandled-error-stack.js:*:*)
    at Module._compile (module.js:*:*)
    at Object.Module._extensions..js (module.js:*:*)
    at Module.load (module.js:*:*)
    at tryModuleLoad (module.js:*:*)
    at Function.Module._load (module.js:*:*)
    at Function.Module.runMain (module.js:*:*)
    at startup (bootstrap_node.js:*:*)
Emitted 'error' event at at:
    at quux (*test-unhandled-error-stack.js:*:*)
    at Object.<anonymous> (*test-unhandled-error-stack.js:*:*)
    at Module._compile (module.js:*:*)
    [... lines matching original stack trace ...]
    at startup (bootstrap_node.js:*:*)
    at bootstrap_node.js:*:*

